ui/src/app/utils/runtime-utils.spec.tsx

```diff
@@ -147,4 +147,23 @@ describe('runtime-utils', () => {
   ])('findMostSevereDiffState(%s) = %s', (diffStates, want) => {
     expect(findMostSevereDiffState(diffStates)).toEqual(want);
   });
+
+  test.each([
```

Comment:
This test failed (every case) in the same way as I see in the UI console: `.get is not a function`